__init__.py

```diff
@@ -273,7 +277,26 @@ def execute(self, context):
         #put a return and paste with indentation
         bpy.ops.text.insert(override, text= '\n'+new)
         return {"FINISHED"}
+        
 
+class DEV_OT_deleteAllDebugPrint(bpy.types.Operator):
+    bl_idname = "devtools.delete_all_debug_print"
+    bl_label = "Delete all debug print"
+    bl_description = "Delete all debug print"
+    bl_options = {"REGISTER"}
+    
+    @classmethod
+    def poll(cls, context):
+        return context.area.type == 'TEXT_EDITOR' #not really necessary the menu is there...
```

Comment:
I have not set `INTERNAL`  in `bl_options` to be able to call through F3 search menu.
Using poll restrict the operator to the context where the F3 search is launched. It avoid possible context error in another area, and also have the benefit to avoid cluttering the search in irrelevant contexts.
So it's still a good idea to specify this.